### DISCUSSION TITLE
Temporal overlapping of stimuli

### DISCUSSION THREAD

**Discussion by pernu822** [jspsych/jsPsych#1545] [Q&A]
Hi I would like to arrange a trial with the following specifications:

Present image1  (jpeg) from t=1 to end of trial
Present audio1 (wave file)  from t =1 to  t =audio1_duration
Record participant audio from t = audio1_duration to t = audio1_duration + 5
Display textbox to receive text input  at  t = audio1_duration + 5   (trial ends after text entry)

I am not sure how to present both image1 and audio1 simultaneously.  And more,  generally how to  temporally overlap two stimuli that that have different start times and end times.

Thanks

Pernu

**Comment by jodeleeuw** [comment]
Hi @pernu822.

You may need to create a custom plugin for this because of the need to keep the image on the screen throughout the trial. 

It might be possible to configure this by using a sequence of plugins: `audio-keyboard-response`, `image-audio-response` (not an official jsPsych plugin), and `survey-text`. In each of those plugins you can probably get the image to display at the same location on the screen by using some clever CSS rules.

**Comment by jdistefano3** [comment]
Hi! I am wanting to do something similar in my experiment (overlapping an image and audio). Have you figured out which plugin can achieve this? Thanks!
